test/maybe/mapTo.test.js

```diff
@@ -0,0 +1,16 @@
+import test from 'ava';
+import { Just, Nothing } from '../../src/Maybe';
+
+test('sets value on the Just instance', t => {
+  const just = Just(10);
+  const newValue = 20;
+
+  t.is(just.mapTo(newValue).getOr(), newValue);
+  t.is(just.mapTo().getOr(), undefined);
```

Comment:
```suggestion
  t.is(just.mapTo().getOr('DEFAULT'), undefined);
```

Чтобы в случае неправильной работы получить diff